tubular/base.py

```diff
@@ -50,7 +50,7 @@ class BaseTransformer(TransformerMixin, BaseEstimator):
 
     """
 
-    def __init__(self, columns=None, copy=True, verbose=False, **kwargs):
+    def __init__(self, columns=None, copy=True, verbose=False):
```

Comment:
Since we have decided to do away with **kwargs, shouldn't the above docstring be updated to not have **kwds Arbitrary keyword arguments.

Reply:
fixed in [10b7c5f](https://github.com/lvgig/tubular/pull/56/commits/10b7c5fae0743ac2c042520a456d3a7aec241207)